src/guides/en/understand-devtools/index.md

```diff
@@ -0,0 +1,236 @@
+---
+date: 2023-11-23
+authors: Patrick Brosset
+title: Understand DevTools
+layout: guide.njk
+---
+
+This guide is an introduction to DevTools, which is a suite of web development tools that are built-in to most web browsers. Read this guide if you are starting web development and haven't used the built-in browser tools yet. You will learn what DevTools is, who it's for, how to open it, how its UI is organized, what tools exist, and what the main user workflows are.
+
+## A word about security first
+
+To some people, DevTools can look complex and sometimes scary. If you've opened DevTools by accident without knowing what it is or understanding web development, it may look like you've been hacked. **Don't worry, this is not the case, DevTools isn't giving privileged access to the website you're viewing, or to your computer, to a bad actor**. You can close the tab where DevTools appeared and continue browsing the web.
+
+Be careful, however, with websites that ask you to open DevTools and, within it, paste code that they provide you. This might be an attempt to steal sensitive information from you. **Never paste code into DevTools unless you understand what the code does**.
+
+## Terminology
+
+DevTools is short for _Developer Tools_.
+
+## How to open DevTools
+
+Before learning more about DevTools, let's open it so you can have it in front of you while reading this guide.
+
+Right-click anywhere on this page and then select **Inspect** (or **Inspect element**, depending on your browser). DevTools opens up in a new browser panel either to the right or at the bottom of the current browser window. You can resize the DevTools panel to be larger or smaller.
+
+If you're using Safari, the **Inspect** menu item isn't visible at first and you need to enabled it.
+
+{% insertTip "enable-safari-devtools" %}
+
+Keep reading to learn more about DevTools.
+
+## What is DevTools
+
+In this guide, we focus on the following web browsers: Firefox, Chrome, Edge, and Safari. However, other browsers such as Opera, Arc, or Brave have similar DevTools. We also focus on the desktop versions of these browsers only. Mobile browsers usually don't have built-in DevTools.
```

Comment:
I know this is a me-thing but a mention of Polypane here wouldn't be out of place, even if just to mention that it's out-of-scope like you mention in the PR.